activerecord/lib/active_record/migration/command_recorder.rb

```diff
@@ -69,7 +69,7 @@ def record(*command, &block)
       # invert the +command+.
       def inverse_of(command, args, &block)
         method = :"invert_#{command}"
-        raise IrreversibleMigration unless respond_to?(method, true)
+        raise IrreversibleMigration, "This migration uses #{command} that is not automatically reversible, please define down method" unless respond_to?(method, true)
```

Comment:
If we suggest defining a `down` method, this means that the user also needs to change `def change` to `def up`.

Another option would be a `reversible do |dir|` block.